### ISSUE TITLE
MNK AoE Combo Adjustment

### ISSUE THREAD

**Issue by someONEgood** [daemitus/XIVComboPlugin#26]
### First, thanks for getting this updated and pushed out so quickly!

**TLDR: Please add another AoE Combo chain (based on Four Point Fury) that maintains the 1,2,3 Combo during Perfect Balance.**

I just wanted to bring to your attention that you may want to add/adjust Monk AoE functionality, the aforementioned being probably the easiest implementation. 

To clarify: For optimal AoE, you cycle between a 1,2,3 Perfect Balance combo, and an X,X,X Perfect Balance Combo.

X,X,X grants Elixer Field and Lunar Nadi = 600 potency
1,2,3 grants Flint Strike and Solar Nadi =600 potency

Once Lunar and Solar Nadi are activated:
Any beast chakra combination during Perfect Balance grants Tornado Kick (850 potency AoE).

This gets even more potent at Lv. 86 (Elixir Field becomes 700 potency Rising Phoenix), and Lv. 90 (Tornado Kick becomes **1000** potency Phantom Rush!).

I think a Four-Point based AoE would be the easiest implementation if you'd be willing, thank you!

**Comment by Grammernatzi** [comment]
The issue I could see with this is that it would be hard to tell what step you would be in at the combo. I don't know if the Monk gauge is finished yet. You could possibly calculate it using Perfect balance stacks.

**Comment by jjgyc** [comment]
Hi there, I'd like to provide some input-  I actually believe that there are 2 ideal implementations (in terms of efficiency, not necessarily ease of implementation).

With 1 being Shadow of the Destroyer, 2 being Four Point Fury and 3 being Rockbreaker- 

Realistically, your aoe combos in perfect balance will always be some variation of 123 (ideally starting always starting with 1), and 111 with everything else being suboptimal.

As such, 

1. Simply having shadow of the destroyer be the base action which leads into the combo chain and have it lock to SOTD during perfect balance (or providing an alternative option for it). 

In terms of player input,  this means that you will be able to only have to press 1 key most of the time- out of PB, you would simply hit SOTD. Inside PB, you would hit SOTD x3, or SOTD > FPF > Rockbreaker if you want the 3 beast chakra option. 

I believe this is likely the easiest implementation.

2. For Rockbreaker to always lead into the combo chain regardless of whether perfect balance or not. This would be the ideal implementation in terms of keyboard space, since you would only need 2 keys - Rockbreaker and SOTD, because you could simply hit rockbreaker x3 for the combo chain normally and in PB, and sotd x3 in PB. 

However, assuming it's difficult to track where the combo is during perfect balance like you mentioned this might be annoying to implement.

**Comment by kaedys** [comment]
> For Rockbreaker to always lead into the combo chain regardless of whether perfect balance or not. This would be the ideal implementation in terms of keyboard space, since you would only need 2 keys - Rockbreaker and SOTD, because you could simply hit rockbreaker x3 for the combo chain normally and in PB, and sotd x3 in PB.
>
> However, assuming it's difficult to track where the combo is during perfect balance like you mentioned this might be annoying to implement.

This is my preference, for the record.  It means you only need two of the 3 on your bar, while your (1) options would require all 3 on your bar, rather defeating the point of the combo consolidation.

As for tracking the combo during PB, I think it could simply be based on either stacks (3 stacks = SotD, 2 stacks = 4PF, 1 stack = RB), or based on the last ability you used.  I think the stack version would probably cause less surprises, though.

**Comment by daemitus** [comment]
Ok, based on copying some work from grammer:

Rockbreaker base:
- if chakra filled (and optional combo enabled)
  - MasterfulBlitz
- if PB/FF active
  - SotD or Rockbreaker depending on level.
- else
  -  Opo > Shadow/Arm
  - Raptor > FpF
  - Coeurl > Rockbreaker
 
FpF base:
- if PB active (should formless be added?)
  - no opo chakra > shadow/arm
  - no raptor chakra > fpf
  - no coeurl chakra > rock
  - otherwise shadow/arm
- else
  - fpf

**Comment by daemitus** [comment]
Open up a new ticket if this needs tweaking. Take a look at `1.3.0.23`
